dbapi_opentracing/tracing.py

```diff
@@ -113,6 +211,24 @@ def _traced_execution(self, function, *args, **kwargs):
             span.set_tag('db.rows_produced', self.rowcount)
         return val
 
+    def __enter__(self):
+        return self
+
+
+class Cursor(_Cursor, wrapt.ObjectProxy):
+    """A wrapper for a DB API Cursor object with traced execute(), executemany(), and callproc() methods."""
+
+    def __init__(self, cursor, tracer=None, span_tags=None, trace_execute=True, trace_executemany=True,
+                 trace_callproc=True, *args, **kwargs):
+        wrapt.ObjectProxy.__init__(self, cursor)
+        _Cursor.__init__(self, tracer, span_tags, trace_execute, trace_executemany, trace_callproc)
+
+    def _get_statement(self, args):
+        return args[0].split(' ')[0]
```

Comment:
Is this guaranteed to not blow up on all the statements we might encounter?

Reply:
No, and I've updated to ensure that what is provided, as long as it's a text type, should be acceptable for us.  Implementations that use custom queries/commands would need to be addressed individually, as I'm not familiar with any.